lib/ipmi_backend_utils.pm

```diff
@@ -243,4 +260,13 @@ sub set_serial_console_on_vh {
 
 }
 
+# reset ipmi main board via mc reset cmd
+sub do_ipmi_mc_reset {
+    die "You should only do mc reset when backend is ipmi!" unless check_var('BACKEND', 'ipmi');
+
+    bmwqemu::diag("Start do_ipmi_mc_reset in test code!!!");
+    console('sol')->do_mc_reset;
```

Comment:
don't merge this before osd has been deployed with your backend changes (most likely 2019)